### DISCUSSION TITLE
Can't add the expense still

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1645] [Q&A] [answered]
Hi,
Can you please check where I did the mistake,
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/e77907c1-50ee-4dd8-9d8f-cf1df41f48ec)

54.227.108.168 - frontend publice IP
54.242.128.0 - backend public ip
52.90.138.168 -  mysql IP

**Comment by sreedharm07** [comment, marked as answer]
@omswetha 
in your backend.service i see two onlines change  it
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/86a55c24-418f-4096-9809-1e48116e068b)
